content/news/events-september-2024.md

```diff
@@ -0,0 +1,63 @@
++++
+headline = "ARTICLE"
++++
+
+{{< centered-image src="events-september-2024.png" width="384">}}
+<br/>
+
+<center><i>You can actually play this one! Head <a href="https://gamebanana.com/mods/540377">here</a> to download and try "A Small FOULAB Adventure" - presented to you by FOULAB's aspiring game devs as part of a donation drive.</i></center>
```

Comment:
What do you think about making it target="_blank" to open in a new tab? Keeps a tab on the Foulab site. I think it's common practice to make off-domain links open in a new tab.

Reply:
Good idea. Did this.